storage/kvstore.go

```diff
@@ -170,6 +171,44 @@ func (s *store) TxnDeleteRange(txnID int64, key, end []byte) (n, rev int64, err
 	return n, rev, nil
 }
 
+func (s *store) RangeEvents(key, end []byte, limit, startRev, endRev int64) (evs []storagepb.Event, nextRev int64, err error) {
```

Comment:
Because WatchableKV is built upon KV, and I keep the most of KV interface. The comment is here: https://github.com/coreos/etcd/pull/3507/files#diff-c813fef69835ab8899a434737095d7c3R45